New package: GraphDynamics v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: GraphDynamics
- Repository: https://github.com/Neuroblox/GraphDynamics.jl
- Created by: @MasonProtter
- Version: v0.1.0
- Commit: 3cb3fde920d4f2507a15af9ed23bcd3aa4c5fb05
- Reviewed by: @MasonProtter
- Reference: https://github.com/Neuroblox/GraphDynamics.jl/commit/3cb3fde920d4f2507a15af9ed23bcd3aa4c5fb05#commitcomment-146901250
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd620b96a2c48e8e221172d732e579fbd02efbc06419b4817d06b6b2678b814ea5a0781387b0c833ac6564ffb915fd1c473bc363d5cbff3576e1f08905f348feac8042764ab5bbc6e3a134f481f8935b176da7f99750d1b3a8462c57224df803d56fa8071933c3c93ac2545ea9d4436b4195512369de6f86536e155b4423250c63cddf58313c128d99084ca122a279266755db689e73dcf49ac81ad94b8e64c629a48b50e16beb242468a1cef3ce337166e -->

goerz commented:
> Documentation and API guides coming soon.

Usually, I insist that basic documentation (not necessarily the *full* documentation) is in place at the time of registration. But maybe for a "backend" package, and for a developer "in good standing" there can be some leeway. Although generally, I try to apply a consistent standard to all registrations.

I'll trust that "soon" means "soon" (maybe even within the waiting period?)

[noblock]

goerz commented:
[noblock]

> Is the package specific to molecules?

I think that's "modular", not "molecular" ;-) Not that there might not be a more descriptive name (hard to tell with this little description in the README)

gdalle commented:
Oops my bad. But then yeah I'd like to see some docs before greenlighting something so close to the JuliaGraphs hunting grounds

MasonProtter commented:
[noblock]

This is a package for generating ODEs / SDEs and later other sorts of DEs to describe the dynamics of a graph of subsystems. I think the name is rather appropriate.

JuliaRegistrator committed:
New package: GraphDynamics v0.1.0

UUID: bcd5d0fe-e6b7-4ef1-9848-780c183c7f4c
Repo: https://github.com/Neuroblox/GraphDynamics.jl.git
Tree: cfe778e13aa718ceb0853b3a48faa7bb917a6d1b

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

MasonProtter commented:
[noblock]

I have added some docstrings to public functions and furnished the README with an example showing the package in use. There's still a lot of docs left to be written but I think this is now more than enough to be suitable for the registry.

@gdalle do you have any remaining concerns, or can you add `[noblock]` to your comments?

goerz commented:
[noblock] Definitely looks good to me!

gdalle commented:
[noblock]
Thanks, it looks good to me! Have updated my previous comments
